v5/openapi.json

```diff
@@ -1,8 +1,8 @@
 {
   "openapi": "3.0.3",
   "info": {
-    "version": "5.1.1",
-    "title": "Pinterest REST API (Beta)",
+    "version": "5.3.0",
```

Comment:
We should be clear (in the release notes) that we never shipped 5.2.0 publicly, or we can adjust the version back to 5.2.0 for this release.